STYLE_GUIDE.md

```diff
@@ -221,6 +221,25 @@ h = {a: 1, b: 2}
 Array.new(3) { |i| i + 1 }
 ```
 
+* <a name="style-rescued-variable-name"></a>
+  We prefer to use self-explained variable names in rescued exceptions
```

Comment:
the current cop pushes us to use `error` as a variable for all exceptions and we can't use other self-explained variable names. 

So what do you think about changing it a little bit. e.g 

```
Use `error` as a variable name on processing exceptions. 
```